bricklayer/catalog/file_scanner.py

```diff
@@ -0,0 +1,91 @@
+"""
+    Module to distribute the S3 download over a spark cluster
+    Results end up in a table
```

Comment:
Might also be worth adding a mention that this is useful when the data is highly partitioned to the point of being unable to use the standard methods for loading data from a mounted bucket.